src/test/scripts/helm_install.sh

```diff
@@ -114,7 +114,7 @@ bootstrap_database() {
   # Use the product name for the name of the postgres database, username, and password.
   # These must match the credentials stored in the Secret preloaded into the namespace,
   # which the application will use to connect to the database.
-  helm install -n "${TARGET_NAMESPACE}" --wait \
+  helm install -n "${TARGET_NAMESPACE}" --wait --timeout 15m \
```

Comment:
Minor comment - is it worth turning these timeout values to constants?

Reply:
Not sure if it's worth it, the timeouts are scattered around a few different files, doing it "right" would mean adding it to the `helm_parameters` file and that one is quite messy already IMHO.